packages/main/prod/package.json

```diff
@@ -8,14 +8,22 @@
     "url": "git+https://github.com/vitessce/vitessce.git"
   },
   "license": "MIT",
+  "type": "module",
   "main": "src/index.js",
   "publishConfig": {
-    "main": "dist/index.min.mjs",
-    "browser": "dist/index.min.umd.js"
+    "main": "dist/index.min.js",
+    "module": "dist/index.min.js",
+    "exports": {
+      ".": {
+        "types": "./dist-tsc/index.d.ts",
+        "import": "./dist/index.min.js"
+      }
+    }
   },
   "files": [
+    "src",
     "dist",
-    "src"
+    "dist-tsc"
```

Comment:
Related to #1374 - this PR corrects package.json files to use `type: module` and also removes references to UMD builds from package.json files. Those UMD builds are still in `dist/` but this effectively hides them as a step towards removing them if there are no issues encountered.

As such, I removed the UMD cypress tests and added cypress tests with https://github.com/keller-mark/dynamic-importmap. I have been testing `dynamic-importmap` in other places: it works on [observable](https://observablehq.com/@keller-mark/vitessce-from-unpkg) with the existing ESM builds and on [colab with anywidget](https://colab.research.google.com/drive/14UFOvH8zaVWMcSnKiKtV9wXweqxOT9YJ?usp=sharing)